templates/app/aspnet-core/src/MyCompanyName.MyProjectName.HttpApi.HostWithIds/appsettings.json

```diff
@@ -1,19 +1,19 @@
 {
-  "App": {
-    "SelfUrl": "https://localhost:44305",
-    "ClientUrl": "http://localhost:4200",
-    "CorsOrigins": "https://*.MyProjectName.com,http://localhost:4200,https://localhost:44307",
-    "RedirectAllowedUrls": "http://localhost:4200,https://localhost:44307"
-  },
-  "ConnectionStrings": {
-    "Default": "Server=(LocalDb)\\MSSQLLocalDB;Database=MyProjectName;Trusted_Connection=True;TrustServerCertificate=True"
-  },
-  "AuthServer": {
-    "Authority": "https://localhost:44305",
-    "RequireHttpsMetadata": "false",
-    "SwaggerClientId": "MyProjectName_Swagger"
-  },
-  "StringEncryption": {
-    "DefaultPassPhrase": "gsKnGZ041HLL4IM8"
-  }
+    "App": {
+        "SelfUrl": "https://localhost:44305",
+        "ClientUrl": "http://localhost:4200",
+        "CorsOrigins": "https://*.MyProjectName.com,http://localhost:4200,https://localhost:44307",
+        "RedirectAllowedUrls": "http://localhost:4200,https://localhost:44307"
+    },
+    "ConnectionStrings": {
+        "Default": "Server=özer\\OZER;Database=MyProjectName;Trusted_Connection=True;TrustServerCertificate=True"
```

Comment:
Should you revert that? we don't want to change connection string with your server name :)